internal/driver/driver.go

```diff
@@ -342,3 +343,9 @@ func valueExtractor(ix int) sampleValueFunc {
 		return v[ix]
 	}
 }
+
+func addPseudoLocs(p *profile.Profile, cfg config, ui plugin.UI) {
```

Comment:
nit: is abbreviating Locations to Locs common in the pprof code? Just something to check for consistency.

Reply:
For function names, it seems abbreviation is not in use.  For variable names, the abbreviated usage exists in multiple files.  I have updated the function names to use the full form.